wolfcrypt/test/test.c

```diff
@@ -22337,19 +22337,20 @@ static int ecc_test_buffers(void)
     TEST_SLEEP();
 
     XMEMSET(plain, 0, sizeof(plain));
+    verify = 1;
```

Comment:
This test case should be a valid one. The variable `verify` should init to 0. And the result check below should be if failed then error out so `if (verify != 1)`.

Reply:
Yeah this whole test confused me as to how it evolved and what it should be. Any idea about setting the plain back to `0`?